build.assets/docker/os.dockerfile

```diff
@@ -9,32 +9,23 @@ ADD os-rootfs/ /
 
 RUN set -ex; \
 	if ! command -v gpg > /dev/null; then \
-	apt-get update; \
+    apt-get update; \
 	apt-get install -y --no-install-recommends \
 	gnupg2 \
 	dirmngr \
 	; \
 	rm -rf /var/lib/apt/lists/*; \
 	fi
 
-# dockerproject debian repo key
-RUN apt-get update && apt-get -q -y install apt-transport-https
-
-RUN (echo 'deb http://httpredir.debian.org/debian/ stretch contrib non-free' >> /etc/apt/sources.list && \
```

Comment:
I don't know why we add those sources here, because of default sources.list contains all that we need. I just updated that list with adding `non-free contrib` in lines below.